crates/bevy_core_pipeline/src/core_2d/camera_2d.rs

```diff
@@ -17,12 +19,25 @@ use bevy_render::{
 };
 use bevy_transform::prelude::{GlobalTransform, Transform};
 
+/// A 2D camera component. Enables the 2D render graph for a [`Camera`].
 #[derive(Component, Default, Reflect, Clone, ExtractComponent)]
 #[extract_component_filter(With<Camera>)]
 #[reflect(Component, Default)]
+#[require(
```

Comment:
SyncToRenderWorld seems to not be added now (as far as I can tell not through any dependencies either). Is this intentional?

Reply:
Good catch, added it for `Camera` now. This wasn't a thing when I did this migration originally, and I forgot to add it when pulling latest changes from `main` :p